addons/mail/static/src/js/emojis_mixin.js

```diff
@@ -9,8 +9,7 @@ var emojis = require('mail.emojis');
  *
```

Comment:
This commit message states "use template instead of JQuery rendering"

Not sure what that means?